presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -563,20 +563,20 @@ Procedures
 
   Create an empty partition in the specified table.
 
-* ``system.sync_partition_metadata(schema_name, table_name, mode, case_sensitive)``
+* ``system.sync_partition_metadata(schema_name, table_name, mode, optional case_sensitive)``
 
   Check and update partitions list in metastore. There are three modes available:
 
   * ``ADD`` : add any partitions that exist on the file system, but not in the metastore.
   * ``DROP``: drop any partitions that exist in the metastore, but not on the file system.
   * ``FULL``: perform both ``ADD`` and ``DROP``.
 
-  The ``case_sensitive`` argument is optional. The default value is ``true`` for compatibility
+  Default value for optional ``case_sensitive`` argument is ``true`` for compatibility
```

Comment:
I am not an expert with articles, but It seems `The default value` is more correct.

Reply:
correct. 

The default vale for the optional ``case_sensitive`` argument is ``true`` for..